paramiko/hostkeys.py

```diff
@@ -25,11 +25,11 @@
 
 from paramiko.py3compat import b, u, encodebytes, decodebytes
 
-try:
-    from collections import MutableMapping
-except ImportError:
+try:                                                    # pragma: no cover
+    from collections import MutableMapping              # pragma: no cover
+except ImportError:                                     # pragma: no cover
     # noinspection PyUnresolvedReferences
-    from UserDict import DictMixin as MutableMapping
+    from UserDict import DictMixin as MutableMapping    # pragma: no cover
```

Comment:
To be clear, I have no authority here, just trying to offer my opinion (to perhaps help refine things or move them along), without being annoying :)